docs/course_materials/linux_intro.html

```diff
@@ -515,6 +515,42 @@ <h3><b>clear</b></h3>
           <section data-markdown="exercises/01_Linux/class_exercise_2.md" data-separator="^\n\n\n" data-separator-vertical="^\n\n">
           </section>
         </section>
+        <section>
+          <section>
+            <h3><b>Vim</b></h3>
+            <ul>
+              <li>Vim is an editor to create or edit a text file.</li>
+              <li>There are two modes in vim: command mode and insert mode.</li>
+              <ul>
+                <li>In the command mode, user can move around the file, delete text, etc.</li>
+                <li>In the insert mode, user can insert text.</li>
+              </ul>
+            </ul>
+          </section>
+          <section>
```

Comment:
Thanks @yifatmakias 
can you add another slide for how to create a file with vim and how to open an existing file with vim.